builder/make_list_build_artifacts

```diff
@@ -11,8 +11,8 @@ artifacts=(".build/$cname-$COMMIT.tar" ".build/$cname-$COMMIT.release" ".build/$
 
 for feature in "${features[@]}"; do
 	for i in "features/$feature/"{image,convert}.*; do
-		extension="$(grep -E -o '(\.[a-z][a-zA-Z0-9\-_]*)*$' <<< "$i")"
-		artifacts+=(".build/$cname-$COMMIT$extension")
+		extension="$(grep -E -o '(\.[a-z][a-zA-Z0-9\-_~]*)*$' <<< "$i")"
```

Comment:
Also, I think the usage of here-string should be stated, not everyone is familiar with this concept of bash. Someone could misread this and only thing of `<<` instead of three. Maybe we could make this a bit more obviously?